index.js

```diff
@@ -44,6 +44,8 @@ Pjax.prototype = {
 
   parseDOM: require("./lib/proto/parse-dom.js"),
 
+  refresh: require("./lib/proto/refresh.js"),
```

Comment:
There is also lib/refresh.js. I am ok for having this refresh here for now, but we should rename the old lib/refresh (and its call) as reload?